_drgn.pyi

```diff
@@ -1165,6 +1165,11 @@ class Type:
     Whether this type takes a variable number of arguments. This is only
     present for function types.
     """
+
+    template_parameters: Sequence[TypeTemplateParameter]
+    """
+    List of template parameters making up this type.
```

Comment:
Maybe "List of template parameters of this type"? This should also mention what types it's valid for (structure, union, class, and function?).